scripts/webpack/postCssPlugins.js

```diff
@@ -22,7 +22,12 @@ const postCssFlexbugsFixes = require('postcss-flexbugs-fixes');
 module.exports = () => [
   postCssFlexbugsFixes,
   autoprefixer({
-    browsers: ['>1%', 'last 4 versions', 'Firefox ESR', 'not ie < 9'],
+    overrideBrowserslist: [
```

Comment:
To remove this warning when building.

![Screen Shot 2020-02-27 at 16 51 10](https://user-images.githubusercontent.com/73652/75466699-7d3d2080-5982-11ea-866d-141043ade9ab.png)